pos_disable_pricelist_selection/__manifest__.py

```diff
@@ -7,6 +7,7 @@
     "license": "LGPL-3",
     "category": "Purchase",
     "website": "https://github.com/OCA/pos",
+    "maintainers": ["ilyasprogrammer"],
     "depends": ["point_of_sale"],
```

Comment:
**chore**: missing dependency on `web_domain_field`

Reply:
web_domain_field is not required. Turns out compute domain thing works without web_domain_field. Just need to use Binary field.

Reply:
Huh, that's good to know. Ok then!